app/controllers/telegram_webhooks_controller.rb

```diff
@@ -9,74 +9,43 @@ def start(*)
     reply_with :message, text: 'Hi there!'
   end
 
-  def shows(*args)
-    endingDate = Date.parse(END_DATE_STRING, t(:date_format))
-
+  def shows(* args)
```

Comment:
@filipecosta01 Is there any specific reason for this space between the star symbol and the args?